plugins/equipment-lock

```diff
@@ -0,0 +1,2 @@
+repository = https://github.com/Matimatician/Equipment-Lock.git
+commit = a5259f4dbd00f5673ec58617c05695b67cafbe39
```

Comment:
```suggestion
repository=https://github.com/Matimatician/Equipment-Lock.git
commit=a5259f4dbd00f5673ec58617c05695b67cafbe39
```

Additionally, I have not reviewed this plugin at all, but if you are consuming clicks/causing people to not be able to equip armour, please make it known to your users in some way that the cause is this plugin. That way you can prevent prevent RL support traffic. You could for example send a game message ``Equipment Lock plugin: unable to equip item because...`` with a cooldown between said messages.

Reply:
Hi, I know you said you haven't reviewed the code yet, but I just wanted you to know that this is already a feature. here's the bit of code from when the event is consumed:

client.addChatMessage(ChatMessageType.GAMEMESSAGE, "", "Someone else in your group has claimed the right to this item, so you cannot equip it.", null);
				event.consume();

Reply:
To YvesW's point, that message doesn't clarify what plugin it comes from. Somehow mentioning "Equipment Lock" will give users looking to figuring it out something to search for. maybe adding `[Equipment Lock] ` in front of messages from the plugin too?

Reply:
understood. I've updated the code to start the game message with "[Equipment Lock]:" to clarify the cause of the consumed action to users.